adserver/analyzer/models.py

```diff
@@ -55,6 +56,8 @@ class AnalyzedUrl(TimeStampedModel):
         ),
     )
 
+    embedding = VectorField(dimensions=384, default=None, null=True, blank=True)
```

Comment:
I suspect we will need some sort of [approximate index here](https://github.com/pgvector/pgvector-python?tab=readme-ov-file#django), but that can come later.

Reply:
Aye, that's definitely a next step once we start querying it.